src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashBuildTrigger.java

```diff
@@ -86,6 +87,7 @@ public StashBuildTrigger(
             boolean checkMergeable,
             boolean mergeOnSuccess,
             boolean checkNotConflicted,
+            boolean checkProbeMergeStatus,
```

Comment:
👎 as this breaks backwards compatibility. Instead it would be best to create a new constructor with the newly added parameter.

Reply:
Thanks for the review, posting an update, hope I got it right (not a Java programmer). I wonder if it is possible for one constructor to call another (reduce duplication), or is that feature only reserved for normal functions?

Reply:
Re-fixed differently after some more suggestions on IRC.